components/bpmn/bpmn-rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/UserService.java

```diff
@@ -42,6 +43,12 @@
 @Path("/userServices/")
 public class UserService {
     private static final Log log = LogFactory.getLog(UserService.class);
+    int tenantId = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId();
+    String str = String.valueOf(tenantId);
+    String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+
+    private static final String DOMAIN_OF_SUPER_TENANT = "carbon.super";
```

Comment:
Use MultitenantConstants.SUPER_TENANT_DOMAIN_NAME instead of re-defining the value